src/libs/ReportUtils.ts

```diff
@@ -7451,7 +7451,7 @@ function isEmptyReport(report: OnyxEntry<Report>): boolean {
     }
 
     // Get the `isEmpty` state from cached report attributes
-    const attributes = reportAttributes?.[report.reportID];
+    const attributes = reportAttributesDerivedValue?.[report.reportID];
```

Comment:
this was renamed here but this spot was added in another PR https://github.com/Expensify/App/pull/61586/files#diff-577fcc5f3a5e4930916ff310e61c38edfa093792fe3ab55b9a30d2aecd8811dbR1061